samples/azure/azure-container-apps-deploy.sh

```diff
@@ -7,7 +7,7 @@ set -euo pipefail
 FILE=".env"
 if [[ -f $FILE ]]; then
 	echo "loading from .env"
-    export $(egrep . $FILE | xargs -n1)
+  eval $(egrep "^[^#;]" $FILE | tr '\n' '\0' | xargs -0 -n1 | sed 's/^/export /')
```

Comment:
it would help to understand what this helps with here.
Bing suggests this which would be a helpful comment:
> The result is a series of commands like export variable=value, which are then evaluated by the eval command and executed by the shell